jigna/js/build.js

```diff
@@ -68,3 +69,37 @@ concatenate({
         'jigna'
     ]
 });
+
+// jigna-vue.js
+concatenate({
```

Comment:
No, that simply attaches a function to be executed on the app run. Nothing actually starts running until we do angular.bootstrap or use ng-app.